docs/software/integrations/mqtt/index.mdx

```diff
@@ -6,21 +6,39 @@ sidebar_position: 6
+The Meshtastic project provides a public MQTT service that users can connect to, with certain restrictions in place to ensure network stability. This service allows Meshtastic devices to bridge over the internet, providing global connectivity for remote networks.
 
-You can also specify your own private MQTT broker to bridge mesh networks together, via the internet (or just a local IP network).
+For instructions on connecting to the public MQTT server, please refer to [Connect to the Default Public Server](/docs/configuration/module/mqtt/#connect-to-the-default-public-server).
 
-You can find the settings available for MQTT [here](/docs/configuration/module/mqtt).
+### Restrictions on the Public MQTT Server
 
-:::important
-When MQTT is turned on, you are potentially broadcasting your entire mesh's traffic onto the public internet. This includes messages and position information.
+To maintain optimal performance and protect LoRa meshes, traffic restrictions are currently applied to the public MQTT server.
 
-The default channel (LongFast) on the public MQTT server usually has a lot of traffic. Your device may get overloaded and may no longer function properly anymore. It is recommended to use a different channel or to use your own MQTT server if you experience issues.
-:::
+#### Zero-Hop Policy
+
+Traffic from the public MQTT server does not fully propagate through local mesh networks. Directly connected nodes will receive the data, but due to the zero-hop policy, it will not spread further to other nodes within the local mesh network.
+
+#### Optimized Traffic Filtering
+
+Only specific portnums are prioritized for transmission over the public MQTT server when using the default PSK:
+
+- NodeinfoApp
+- TextMessageCompressedApp
+- TextMessageApp
+- PositionApp
+- TelemetryApp
+- MapReportApp
+- RoutingApp
+
+This filtering focuses network resources on critical traffic, improving overall performance and reducing unnecessary data flow. Since these restrictions are applied at the network level, no firmware updates are required. As Meshtastic networks continue to grow, further traffic reduction measures may become necessary to manage network load and maintain reliable performance across all channels.
+
+### Using Private Brokers
+
+It is not recommended to use the default key (PSK) on a private broker. Doing so introduces security vulnerabilities and can flood the mesh with traffic, as private brokers do not enforce the zero-hop policy needed for public channels. Private brokers are intended for use with private channels, where custom PSKs provide secure, isolated communication.
```

Comment:
Minor quibble: I would phrase it "potentially allows security vulnerabilities" because it doesn't inherently introduce them, but there is no traffic filtering safeguard in place